### PR TITLE
Add Search Nodes Description in the Creating a Cluster Page

### PR DESCRIPTION
**Pull request by augustoliks**
### Search Node Description

There are no documented `search` Node type in the [tuning-your-cluster/index/](https://opensearch.org/docs/2.8/tuning-your-cluster/index/) Page. 

_In my commit, I created a description based on [Searchable Snapshots Feature Blog Post](https://opensearch.org/blog/searchable-snapshots/), and in the Documentation Page [Searchable Snapshots](https://opensearch.org/docs/2.8/tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot/)_

> drawing
>
>> Link: https://opensearch.org/docs/2.8/tuning-your-cluster/index/

---

### Fix the Searchable Snapshot Snippet Example 

I found an error in the Searchable Snapshot example. It is a simple tabulation error.

> drawing
>
>> Link: https://opensearch.org/docs/2.8/tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot/

### CODE REVIEW

**Review by andrross — APPROVED** [review]

File: _tuning-your-cluster/index.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ Cluster manager eligible | Elects one node among them as the cluster manager nod
...
 Coordinating | Delegates client requests to the shards on the data nodes, collects and aggregates the results into one final result, and sends this result back to the client. | A couple of dedicated coordinating-only nodes is appropriate to prevent bottlenecks for search-heavy workloads. We recommend using CPUs with as many cores as you can.
+Search  | Provides access to [Searchable Snapshots]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot/). Incorporates techniques like caching frequently used data segments and removing the least used to access index `remote_snapshot` (stored in remote long-term storage source, for example AWS S3 or GCS). | The index allocated in Search nodes is a cache to Snapshots. For this reason, we recommend dedicated nodes with a setup that privileges more computing (CPUs and Memory) than large space storage (Hard Disk).
```
Comment (kolchfa-aws) [inline _tuning-your-cluster/index.md:37]:
```suggestion
Search  | Provides access to [searchable snapshots]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot/). Incorporates techniques like caching frequently used and removing the least used data segments to access the searchable snapshot index (stored in remote long-term storage source, for example AWS S3 or Google Cloud Storage). | The index allocated in search nodes is a cache to snapshots. Thus, we recommend dedicated nodes with a setup that prefers more computing (CPUs and memory) than large space storage (hard disk).
```

**Review by kolchfa-aws — COMMENTED** [review]
@augustoliks A small rewording suggestion for clarity. Thanks.

File: _tuning-your-cluster/index.md (hunk 1 above)
Comment (augustoliks) [reply _tuning-your-cluster/index.md]: adopted!

**Review by natebower — APPROVED** [review]
LGTM with noted changes.

**Review by kolchfa-aws — APPROVED** [review]
LGTM. Thanks, @augustoliks! Let's get this merged.

File: _tuning-your-cluster/index.md [hunk 2]
```diff
@@ -34,6 +34,7 @@ Cluster manager eligible | Elects one node among them as the cluster manager nod
...
 Coordinating | Delegates client requests to the shards on the data nodes, collects and aggregates the results into one final result, and sends this result back to the client. | A couple of dedicated coordinating-only nodes is appropriate to prevent bottlenecks for search-heavy workloads. We recommend using CPUs with as many cores as you can.
+Search  | Provides access to [searchable snapshots]({{site.url}}{{site.baseurl}}/tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot/). Incorporates techniques like caching frequently used and removing the least used data segments to access the searchable snapshot index (stored in remote long-term storage source, for example AWS S3 or Google Cloud Storage). | The index allocated in search nodes is a cache to snapshots. Thus, we recommend dedicated nodes with a setup that prefers more computing (CPUs and memory) than large space storage (hard disk).
```
[the remark this answers is not in the document]
Comment (augustoliks) [reply _tuning-your-cluster/index.md]: adopted! :)

**Pull request merged**